corehq/apps/app_manager/models.py

```diff
@@ -2416,22 +2422,34 @@ class ModuleBase(IndexedSchema, NavMenuItemMediaMixin, CommentMixin):
     def is_surveys(self):
         return self.case_type == ""
 
+    def after_add(self):
```

Comment:
Maybe just a style difference, but I think `init` or `initialize` would be a more typical name than `after_add`

Reply:
would that work in `init`? I think this needs to be done once all things are set so added as something that should be run each time a module is added.

Reply:
Sorry, that was a confusing comment. I just meant the naming, forgot that python has a built in `__init__` so naming another function `init` is a bad idea. Ignore this.

Reply:
ya, i was a bit hesitant about that name at least. got this from my days working on ruby on rails where this is a "after_" naming convention.
For now, It's basically this should run after each wrap of the document or every time a module is added. Can't think of a way where this can be actually hooked in the object life cycle itself.